Search/Type/BooleanType.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * (c) SimpleThings GmbH <info@simplethings.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SimpleThings\Bundle\SolrBundle\Search\Type;
+
+use SimpleThings\Bundle\SolrBundle\Metadata\PropertyMetadata;
+use SimpleThings\Bundle\SolrBundle\Search\Field\SchemaField;
+
+/**
+ * @author Tobias Gödderz <tg@simplethings.de>
+ */
+class BooleanType extends Type
+{
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function convertValue($value)
+    {
+        return boolval($value);
```

Comment:
`return (bool)$value;`

Reply:
@DavidBadura Hatte ich der Konsistenz halber so gemacht, weil in den anderen Types auch `intval()` statt `(int)` etc. verwendet wird. Soll ich das hier trotzdem in diesem Pull Request ändern?

Reply:
Ja, wäre am besten. Man kann das bei den anderen auch anpassen ;-)